core/src/main/java/io/aiven/klaw/service/ServerConfigService.java

```diff
@@ -460,6 +494,29 @@ private boolean validateTenantConfig(TenantConfig dynamicObj, int tenantId) {
     return tenantCheck;
   }
 
+  private void isBaseSyncValid(String baseSync, List<String> existingResources)
+      throws KlawException {
+    if (baseSync != null && !existingResources.contains(baseSync)) {
+      throw new KlawException(
```

Comment:
Currently we are sending an ApiResponse object in case of any errors and we display that error. Not sure throwing exception is being displayed in front end.

Reply:
Correct here, at the moment what Is happening is we log the error and then return false like the previous validation.
I could change this to re-throw the error but it just changes the current experience the user gets on the front end.
I'll test what this looks like now and we can decide if that is an avenue we want to persue.

Reply:
Example output from returning more detailed error to the UI. 
We will go with this for this PR.
![image](https://user-images.githubusercontent.com/121855584/212289572-2cf30bd4-4a42-466c-8276-b25a7efdca52.png)